packages/client/src/locale/en.tsx

```diff
@@ -0,0 +1,15 @@
+import LocalizedStrings from 'react-localization';
+
+let strings = new LocalizedStrings({
```

Comment:
I am pro localisation, how's the TypeScript support in this package?

Reply:
https://github.com/stefalda/react-localization/blob/master/README.md#typescript-support

Reply:
haha oh my god how did I not see that